api/lanes.py

```diff
@@ -1,401 +1,769 @@
+        ya_fiction.sublanes.append(ya_fiction_best_sellers)
+
+    ya_fiction.sublanes.append(
+        lane_from_genres(_db, library, [genres.Dystopian_SF],
+                         identifier=ya_fiction_identifier + " " + genres.Dystopian_SF.name,
+                         priority=ya_fiction_priority, **ya_common_args))
+    ya_fiction_priority += 1
+    ya_fiction.sublanes.append(
+        lane_from_genres(_db, library, [genres.Fantasy],
+                         identifier=ya_fiction_identifier + " " + genres.Fantasy.name,
+                         priority=ya_fiction_priority, **ya_common_args))
+    ya_fiction_priority += 1
+    ya_fiction.sublanes.append(
+        lane_from_genres(_db, library, [genres.Comics_Graphic_Novels],
+                         identifier=ya_fiction_identifier + " " + genres.Comics_Graphic_Novels.name,
+                         priority=ya_fiction_priority, **ya_common_args))
+    ya_fiction_priority += 1
+    ya_fiction.sublanes.append(
+        lane_from_genres(_db, library, [genres.Literary_Fiction],
+                         identifier=ya_fiction_identifier + " " + genres.Literary_Fiction.name,
+                         display_name="Contemporary Fiction",
+                         priority=ya_fiction_priority, **ya_common_args))
+    ya_fiction_priority += 1
+    ya_fiction.sublanes.append(
+        lane_from_genres(_db, library, [genres.LGBTQ_Fiction],
+                         identifier=ya_fiction_identifier + " " + genres.LGBTQ_Fiction.name,
+                         priority=ya_fiction_priority, **ya_common_args))
+    ya_fiction_priority += 1
+    ya_fiction.sublanes.append(
+        lane_from_genres(_db, library, [genres.Suspense_Thriller, genres.Mystery],
+                         identifier=ya_fiction_identifier + " Mystery & Thriller",
+                         display_name="Mystery & Thriller",
+                         priority=ya_fiction_priority, **ya_common_args))
+    ya_fiction_priority += 1
+    ya_fiction.sublanes.append(
+        lane_from_genres(_db, library, [genres.Romance],
+                         identifier=ya_fiction_identifier + " " + genres.Romance.name,
+                         priority=ya_fiction_priority, **ya_common_args))
+    ya_fiction_priority += 1
+    ya_fiction.sublanes.append(
+        lane_from_genres(_db, library, [genres.Science_Fiction],
+                         exclude_genres=[genres.Dystopian_SF, genres.Steampunk],
+                         identifier=ya_fiction_identifier + " " + genres.Science_Fiction.name,
+                         priority=ya_fiction_priority, **ya_common_args))
+    ya_fiction_priority += 1
+    ya_fiction.sublanes.append(
+        lane_from_genres(_db, library, [genres.Steampunk],
+                         identifier=ya_fiction_identifier + " " + genres.Steampunk.name,
+                         priority=ya_fiction_priority, **ya_common_args))
+    ya_fiction_priority += 1
+    # TODO:
+    # Paranormal -- what is it exactly?
```

Comment:
I think we can get rid of this TODO. We have not had a groundswell of complaints that there's no separate "Paranormal" lane.